src/Components/Inquiry/Views/InquiryBasicInfo.tsx

```diff
@@ -18,158 +17,146 @@ import {
-        placeholder="Memberships, institutions, positions"
-        onChange={handleInputChange("otherRelevantPositions")}
-        defaultValue={me?.otherRelevantPositions ?? undefined}
-        mb={2}
-      />
-
-      <Button
-        type="submit"
-        width="100%"
-        loading={mode === "Loading"}
-        disabled={mode === "Success"}
-      >
-        Next
-      </Button>
-    </Box>
+    <Formik<InquiryBasicInfoInput>
+      onSubmit={handleSubmit}
+      initialValues={{
+        name: me?.name ?? "",
+        location: me?.location ?? {},
+        profession: me?.profession ?? "",
+        otherRelevantPositions: me?.otherRelevantPositions ?? "",
+      }}
+    >
+      {({ values, setFieldValue, handleChange, handleBlur, isSubmitting }) => {
+        return (
+          <Stack as={Form} gap={4}>
+            <Stack gap={2}>
+              <Text variant="lg-display" pr={2}>
+                Tell {artwork.partner?.name ?? "us"} a little bit about
+                yourself.
+              </Text>
+
+              <Input
+                title="Name"
+                name="name"
+                defaultValue={me?.name ?? ""}
+                value={values.name}
+                onChange={handleChange}
+                onBlur={handleBlur}
+                required
+              />
+
+              <LocationAutocompleteInput
+                title="Primary Location"
+                name="location"
+                placeholder="Location"
+                maxLength={256}
+                spellCheck={false}
+                defaultValue={me?.location?.display ?? ""}
+                onChange={place => {
+                  setFieldValue("location", normalizePlace(place))
+                }}
+              />
+
+              <Input
+                title="Profession"
+                name="profession"
+                placeholder="Profession"
+                value={values.profession}
+                onChange={handleChange}
+                onBlur={handleBlur}
+              />
+
+              <Input
+                title="Other relevant positions"
+                name="otherRelevantPositions"
+                placeholder="Memberships, institutions, positions"
+                value={values.otherRelevantPositions}
+                onChange={handleChange}
+                onBlur={handleBlur}
+              />
+            </Stack>
+
+            <Button type="submit" width="100%" loading={isSubmitting}>
+              Save & Continue
+            </Button>
+          </Stack>
+        )
+      }}
+    </Formik>
   )
 }
 
 const InquiryBasicInfoPlaceholder: React.FC = () => {
   return (
     <Skeleton>
-      <SkeletonText variant="lg-display">
-        Tell Example Partner a little bit about yourself.
-      </SkeletonText>
+      <Stack gap={4}>
+        <Stack gap={2}>
+          <SkeletonText variant="lg-display">
+            Tell Example Partner a little bit about yourself.
+          </SkeletonText>
```

Comment:
I'm surprised to see that this form has a skeleton loader since it's so small. I'm building this form in Eigen at the moment. Do you think I should include it there too?

Reply:
If there's something that loads, it needs a skeleton. In this case we are loading the default values. You may have access to them already in your case. If not, then loader, yeah.